modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -170,7 +170,8 @@ inline GRunArgsP& operator += (GRunArgsP &lhs, const GRunArgsP &rhs)
 namespace gapi
 {
     GAPI_EXPORTS cv::GRunArgsP bind(cv::GRunArgs &results);
-} // namespace gapi
+    GAPI_EXPORTS cv::GRunArg   bind(cv::GRunArgP &out);     // FIXME: think more about it
```

Comment:
I see this part is already integrated by @mpashchenkov 

Can you please sync on this?

Reply:
It wasn't merged, so I closed that one, since I needed the changes here